microsoft/rnx-kit
# refactor(align-deps): migrate set-version command

### Description

Migrates the `set-version` command.

### Test plan

As with the `init` command, the `set-version` is currently also only available for React Native packages.

- Verify that `align-deps` fails if the configuration is missing:
  ```
  cd packages/babel-preset-metro-react-native
  node ../align-deps/lib/cli.js --set-version 0.68
  ```
- Verify that `align-deps` fails if the configuration is incomplete:
  ```
  cd packages/metro-config
  node ../align-deps/lib/cli.js --set-version 0.68
  ```
- `align-deps` will migrate while updating requirements:
  ```
  cd packages/test-app
  node ../align-deps/lib/cli.js --set-version 0.68
  ```
